src/sequence.jl

```diff
@@ -241,19 +241,23 @@ end
 function Base.showcompact(io::IO, seq::Sequence)
     # don't show more than this many characters
     # to avoid filling the screen with junk
-    width = displaysize()[2]
-    if length(seq) > width
-        half = div(width, 2)
-        for i in 1:half-1
-            print(io, seq[i])
-        end
-        print(io, '…')
-        for i in endof(seq)-half+2:endof(seq)
-            print(io, seq[i])
-        end
+    if length(seq) == 0
+        print(io, "< EMPTY SEQUENCE >")
```

Comment:
Adding at least one test case would be good.

Reply:
👍 I will do this soon